extensions/amp-analytics/0.1/test/vendor-requests.json

```diff
@@ -450,6 +450,10 @@
     "elementInteract": "https://tags-dmp.vpadn.com/et?t=_timestamp_&sdkn=j&sdkv=1.2.0&lk=!licence_key&en=UTF-8&ctid=_client_id_&ev=element_interact&pl={\"name\":\"!category\",\"action\":\"!action\",\"value\":\"!label\"}&is_amp=1&page_id=_page_view_id_",
     "event": "https://tags-dmp.vpadn.com/et?t=_timestamp_&sdkn=j&sdkv=1.2.0&lk=!licence_key&en=UTF-8&ctid=_client_id_&ev=!ev_name&pl=!payload&is_amp=1&page_id=_page_view_id_"
   },
+  "webengage": {
+    "base": "https://c.webengage.com/amp?licenseCode=!licenseCode&luid=_client_id_&pageUrl=_canonical_url_&pageTitle=_title_&referrer=_document_referrer_&vh=_viewport_height_&vw=_viewport_width_&category=application",
```

Comment:
The requests will be resolved with the `extraUrlParams` you specified. `....&isAmp=1` that will fix the test.